core/backend/src/rpc-impl/RpcBriefcaseUtility.ts

```diff
@@ -112,7 +112,9 @@ export class RpcBriefcaseUtility {
     if (undefined === iModelDb)
       throw new IModelError(IModelStatus.NotOpen, "iModel is not opened", () => iModel);
 
-    await iModelDb.reattachDaemon(accessToken); // just in case this is a V2 checkpoint whose accessToken is about to expire.
+    // call reattach, just in case this is a V2 checkpoint whose accessToken is about to expire.
+    // Note: there's no reason to wait for this to complete.
+    iModelDb.reattachDaemon(accessToken); // eslint-disable-line @typescript-eslint/no-floating-promises

```

Comment:
I'm worried we're missing an edge case here...

Let's say a user has a web backend instance started and I just completely quits their frontend a few minutes before `_reattachTimestamp`.  If the backend just sees radio silence for until another user comes along far after the SAS has expired, then we're potentially in a position where not awaiting `reattachDaemon` means we may use the stale SAS...

Reply:
how does awaiting change anything about whether or not the refresh succeeds?

Reply:
i'll have to all requests await the same promise until the token is refreshed.

Reply:
added await back. All requests will await the first one's promise.